app/allocation/views/view.njk

```diff
@@ -66,7 +78,32 @@
     {% endif %}
   </section>
 
-  <section class="govuk-!-margin-top-8">
+
+
+  {% if canAccess('allocation:person:assign') %}
+    <section class="govuk-!-margin-top-8">
+    <h2 class="govuk-heading-m govuk-!-margin-bottom-1">{{ t("allocations::view.people_assign.heading") }}</h2>
+    {% set slotsEmpty = allocation.moves_count - allocationPeople.filledSlots.length %}
+    {% if slotsEmpty %}
+    <p>You can’t add people who require a special vehicle to an allocation - you need to <a href="/move/new">create a single request</a> instead</p>
+
+    <h3 class="govuk-heading-s">{{ t("allocations::view.people_assign.slots_empty", {
+      slots_empty: slotsEmpty,
+      person_or_people: t('person', {
+        count: slotsEmpty
+      })
+    }) }}</h3>
+      <p>{{ govukButton({
+        html: "Add person",
+        href: "/allocation/" + allocation.id + "/assign"
+      }) }}</p>
+    {% endif %}
+    {% if allocationPeople.filledSlots.length === 0 %}
+      {{ emptySlot('No prisoners added to allocation') }}
```

Comment:
Feel like we have a mismatch between `person` and `prisoner`. I think we should be using `person` where possible.

Reply:
The bigger issue is surely that this text is not in the locales file ;) Thats's how the copy is - am chasing up.

Reply:
I assumed moving it to locales was still the WIP part...